shared_model/validators/validators_common.hpp

```diff
@@ -11,6 +11,17 @@
 namespace shared_model {
   namespace validation {
 
+    /**
+     * A class that contains configuration parameters for all validators.
+     * A validator may read only specific fields.
+     */
+    class ValidatorsConfig {
```

Comment:
a thing with public members is usually called a struct
```suggestion
    struct ValidatorsConfig {
```